src/app/lib/analyticsUtils/chartbeat.js

```diff
@@ -0,0 +1,45 @@
+import Cookie from 'js-cookie';
+import onClient from '../utilities/onClient';
+
+const ID_COOKIE = 'ckns_sylphid';
+
+export const chartbeatUID = 50924;
+
+export const useCanonical = true;
+
+export const getSylphidCookie = () => {
+  if (!onClient()) {
+    return null;
+  }
+
+  return Cookie.get(ID_COOKIE) ? Cookie.get(ID_COOKIE) : null;
+};
+
+export const domain = service => {
```

Comment:
```suggestion
export const getDomain = service => {
```

And change the references to this function